pkg/domain/entities/images.go

```diff
@@ -171,3 +171,16 @@ type ImagePruneReport struct {
 	Report Report
 	Size   int64
 }
+
+type ImageImportOptions struct {
+	Changes     []string
+	Message     string
+	Quiet       bool
+	Reference   string
+	Source      string
+	SourceIsURL bool
+}
+
+type ImageImportReport struct {
+	Id string
```

Comment:
s/Id/ID/ ... for the linter

Reply:
we get into this push and pull between go templates and the linter here. leaving for now.